src/tests/test_init.py

```diff
@@ -0,0 +1,42 @@
+import sys
```

Comment:
With your current setup, everything in the `src` directory will be installed in the user's `site-packages` and added to their search path. So in Python, `import tests` will try to import your test suite. This is not what you want; `tests` should sit outside `src`.

Reply:
Done.

Reply:
The same logic applies to `examples` and `documents`.

Reply:
You are right, I have just corrected this one!